SAKPaaS/src/app/components/user-card/user-card.component.html

```diff
@@ -1,25 +1,31 @@
 <mat-card *ngIf="authService.getUser() | async as user; else login" id="user-card">
-  <mat-card-title>
-    {{user.firstName}}
-    <mat-icon fontSet="material-icons-outlined">
-      settings
-    </mat-icon>
-  </mat-card-title>
-  <mat-card-content>
-    {{user.username}}<br>
-    <a class="mat-primary" (click)="authService.logout()">{{ 'user-card.logout' | translate }}</a><br>
-  </mat-card-content>
+  <div class="expand">
+    <mat-icon (click)="change()" class="expand-icon">keyboard_arrow_left</mat-icon>
+  </div>
+  <div class="content">
+    <mat-card-title>
+      <span *ngIf="expanded">{{ 'user-card.greeting' | translate }}</span> {{user.firstName}}
+      <mat-icon fontSet="material-icons-outlined" class="settings-icon">settings</mat-icon>
+    </mat-card-title>
+    <mat-card-content>
+      {{user.username}}<br>
```

Comment:
Hier vielleicht die Länge begrenzen, damit das nicht aus dem container rausläuft, wenn der Anmeldename (in meinem Fall z.B. die Emailadresse) zu lang ist :)